tamr_unify_client/project/attribute_mapping_collection.py

```diff
@@ -0,0 +1,30 @@
+import requests
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+from tamr_unify_client.project.attribute_mapping import AttributeMapping
+
+
+class AttributeMappingCollection(AttributeMapping):
+    def setUp(self):
+        auth = UsernamePasswordAuth("username", "password")
+        self.unify = Client(auth)
+
+    def stream(self, map_url):
+        map_url = map_url
+        all_maps = requests.get(map_url).json()
+        return all_maps
+
+    def by_resource_id(self, map_url, resource_id):
+        aMaps = self.stream(map_url)
+        length = len(aMaps)
+        for y in range(length):
+            if resource_id in str(aMaps[y]):
+                return aMaps[y]
+
+    def by_relative_id(self, map_url, relative_id):
+        aMaps = self.stream(map_url)
```

Comment:
Since relative ID and resource ID are linked, it makes sense to implement one fully, and then call the other in the second so as not to repeat code